packages/splits-sdk/src/types.ts

```diff
@@ -31,11 +31,27 @@ export type MulticallConfig = {
   calls: CallData[]
 } & TransactionOverridesDict
 
+export type DataClientConfig = {
+  publicClient?: PublicClient<Transport, Chain>
+  apiConfig: {
+    apiKey: string
+    serverURL?: string
```

Comment:
nit: is this for debugging?

Reply:
Our client actually uses this to send request to the non external graphQL endpoint